overlord/snapstate/handlers.go

```diff
@@ -4083,8 +4083,8 @@ func (m *SnapManager) doEnforceValidationSets(t *state.Task, _ *tomb.Tomb) error
 	st.Lock()
 	defer st.Unlock()
 
-	var userID int
-	if err := t.Get("userID", &userID); err != nil {
+	var local bool
+	if err := t.Get("local", &local); err != nil {
```

Comment:
for local I don't think it make sense to get full encoded assertion attached to the task, instead we should just get enough information to find them in the system database in assertstate

Reply:
Yea that makes sense, changed this